### ISSUE TITLE
Uncaught TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by wnitya2** [wallabyjs/atom-quokka#7]
[Enter steps to reproduce:]

1. Create new .js file
2. Ctrl+Shift+P, then select Quokka: Make Quokka From Existing File

Note: the error happened after I update atom-quokka package to version 1.0.59

**Atom**: 1.24.0 x64
**Electron**: 1.6.16
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [atom-quokka](https://github.com/wallabyjs/atom-quokka) package 1.0.59

### Stack Trace

Uncaught TypeError: Cannot read property 'length' of undefined

```
At C:\Users\Nitya\.atom\packages\atom-quokka\node_modules\quokka-atom\build\quokka.js:1

TypeError: Cannot read property 'length' of undefined
    at Function._getFileType (/packages/atom-quokka/node_modules/quokka-atom/build/quokka.js:1:26150)
    at d._start (/packages/atom-quokka/node_modules/quokka-atom/build/quokka.js:1:14580)
    at d.makeQuokkaFromExistingFile (/packages/atom-quokka/node_modules/quokka-atom/build/quokka.js:1:11941)
    at /packages/atom-quokka/node_modules/quokka-atom/build/quokka.js:1:10137)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.24.0/resources/app/src/command-registry.js:388:36)
    at Object.didConfirmSelection (~/AppData/Local/atom/app-1.24.0/resources/app/node_modules/command-palette/lib/command-palette-view.js:165:35)
    at SelectListView.confirmSelection (~/AppData/Local/atom/app-1.24.0/resources/app/node_modules/command-palette/node_modules/atom-select-list/src/select-list-view.js:401:26)
    at HTMLDivElement.core:confirm (~/AppData/Local/atom/app-1.24.0/resources/app/node_modules/command-palette/node_modules/atom-select-list/src/select-list-view.js:106:20)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.24.0/resources/app/src/command-registry.js:388:36)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (~/AppData/Local/atom/app-1.24.0/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (~/AppData/Local/atom/app-1.24.0/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)
    at WindowEventHandler.handleDocumentKeyEvent (~/AppData/Local/atom/app-1.24.0/resources/app/src/window-event-handler.js:108:40)
```

### Commands

```
     -1:16.8.0 core:copy (input.hidden-input)
     -1:14.7.0 intentions:highlight (input.hidden-input)
     -1:14.5.0 core:paste (input.hidden-input)
     -1:13.5.0 core:move-left (input.hidden-input)
  3x -1:13.2.0 core:select-left (input.hidden-input)
     -1:11.5.0 intentions:highlight (input.hidden-input)
     -1:11.3.0 core:save (input.hidden-input)
 16x -0:56.3.0 core:select-right (input.hidden-input)
     -0:30.7.0 tree-view:add-file (span.name.icon.icon-file-directory)
     -0:27.8.0 core:confirm (input.hidden-input)
     -0:26.5.0 intentions:highlight (input.hidden-input)
     -0:26.3.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
     -0:25.2.0 command-palette:toggle (input.hidden-input)
     -0:18.1.0 core:move-down (input.hidden-input)
     -0:17.7.0 core:confirm (input.hidden-input)
     -0:17.7.0 quokka:make-quokka-from-existing-file (input.hidden-input)
```

### Non-Core Packages

```
atom-quokka 1.0.59 
busy-signal 1.4.3 
go-plus 5.8.2 
intentions 1.1.5 
linter 2.2.0 
linter-eslint 8.4.1 
linter-ui-default 1.6.10 
platformio-ide-terminal 2.8.0 
remember-folds 0.3.0 
structure-view 0.2.1 
```

**Comment by ArtemGovorov** [comment]
Not exactly sure what was causing the issue, neither I can reproduce it. Previous versions used to handle the scenario in exact same way, so probably something changed in your environment. Somehow requesting the file (the one you are running Quokka on) for its file type via its grammar fails.

Anyway, we have added added some check to avoid the error if the file type can't be requested, so the issue should be hopefully fixed for you in the published v1.0.61.

**Comment by wnitya2** [comment]
thank you very much!
